src/modules/jcms_rest/src/Plugin/rest/resource/CollectionListRestResource.php

```diff
@@ -38,6 +39,26 @@ public function get() {
 
     $this->filterSubjects($base_query);
 
+    $containing = \Drupal::request()->query->get('containing', []);
+    if (!empty($containing)) {
+      $orCondition = $base_query->orConditionGroup();
+
+      foreach ($containing as $item) {
+        preg_match('~^(article|blog-article|interview)/([a-z0-9-]+)$~', $item, $matches);
+
+        if (empty($matches[1]) || empty($matches[2])) {
+          throw new JCMSBadRequestHttpException(t('Invalid containing parameter'));
+        }
+
+        $orCondition = $orCondition->condition($base_query->andConditionGroup()
+          ->condition('field_collection_content.entity.type', str_replace('-', '_', $matches[1]))
+          ->condition('field_collection_content.entity.uuid', $matches[2], 'ENDS_WITH')
```

Comment:
is it because the `uuid` is the full one in the database (e.g. `de43e8c6-8196-11e7-bb31-be2e44b06b34`) and we are only using its end (e.g. `44b06b34`)? I wonder how duplicates protection works.